docs/specs/markdown.yml

```diff
@@ -403,7 +403,7 @@ outputs:
 # remove host name for external link
 inputs:
   docfx.yml: |
-    hostName: docs.com
+    renderingHostName: docs.com
```

Comment:
This sounds more like `removeHostName` for me 😁

Reply:
used for `xrefmap` as well though 😂

Reply:
Add `RemoveHostName` along with `XrefHostName`